tests/dash/test_dash_privatelink.py

```diff
@@ -0,0 +1,73 @@
+import logging
+from ipaddress import ip_interface
+
+import configs.privatelink_config as pl
+import ptf.testutils as testutils
+import pytest
+from constants import LOCAL_PTF_INTF
+from gnmi_utils import apply_messages
+from packets import outbound_pl_packets
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="module")
+def dpu_ip(duthost, dpu_index):
+    cmd = f"ip addr show | grep Ethernet-BP{dpu_index} | grep inet | awk '{{print $2}}'"
+    npu_interface_ip = ip_interface(duthost.shell(cmd)["stdout"].strip())
+    return npu_interface_ip.ip + 1
+
+
+@pytest.fixture(scope="module", autouse=True)
+def add_dpu_static_route(duthost, dpu_ip):
+    cmd = f"ip route replace {pl.SIP}/32 via {dpu_ip}"
+    duthost.shell(cmd)
+
+    yield
+
+    duthost.shell(f"ip route del {pl.SIP}")
```

Comment:
Before yield, it uses replace to change the static route, which means that a route related to pl.sip may exist. So, if we use del to remove the pl.sip route, it seems that it cannot restore to the original route status. Right?

Reply:
We expect no related routes to exist on the switch prior to the test, but use 'replace' here in case there are any routes that weren't cleaned up properly so that they will be overwritten